scripts/Dockerfile.verify

```
@@ -0,0 +1,65 @@
+FROM node:22-slim AS base
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends sudo git ca-certificates && \
+    rm -rf /var/lib/apt/lists/*
+
+# Claude Code blocks root — create non-root user
+RUN useradd -m -s /bin/bash testuser && \
+    echo "testuser ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
+
+# Install Claude Code globally
+RUN npm install -g @anthropic-ai/claude-code
```

Hadolint (2.14.0)

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)